### PR TITLE
HDDS-2454. Improve OM HA robot tests.

### PR DESCRIPTION
**Pull request by hanishakoneru**
## What changes were proposed in this pull request?
In one CI run, testOMHA.robot failed because robot framework SSH commands failed. This Jira aims to verify that the command execution succeeds.

## What is the link to the Apache JIRA
https://issues.apache.org/jira/browse/HDDS-2454

## How was this patch tested?
acceptance test - smoketest/omha/testOMHA.robot

### CODE REVIEW

File: hadoop-ozone/dist/src/main/smoketest/omha/testOMHA.robot [hunk 1]
```diff
@@ -42,17 +42,21 @@ Start OM
...
                             Set Global Variable             ${HOST}                 ${OM_HOST}
                             Open Connection And Log In
-                            Execute Command                 /opt/startOM.sh --restart
-    ${startupMsg} =         Execute Command                 sudo ps aux | grep om
+    ${rc1} =                Execute Command                 /opt/startOM.sh --restart
+                            Should Be Equal As Integers     ${rc1}                  0
+    ${rc2}  ${startMsg} =   Execute Command                 sudo ps aux | grep om
+                            Should Be Equal As Integers     ${rc2}                  0
                             Close Connection
                             Should Contain                  ${startupMsg}           OzoneManagerStarter
```
Comment (adoroszlai) [inline hadoop-ozone/dist/src/main/smoketest/omha/testOMHA.robot]:
```suggestion
                             Should Contain                  ${startMsg}           OzoneManagerStarter
```

File: hadoop-ozone/dist/src/main/smoketest/omha/testOMHA.robot [hunk 2]
```diff
@@ -42,17 +42,21 @@ Start OM
...
                             Set Global Variable             ${HOST}                 ${OM_HOST}
                             Open Connection And Log In
-                            Execute Command                 /opt/stopOM.sh
-    ${shutdownMsg} =        Execute Command                 sudo ps aux | grep om
+    ${rc1} =                Execute Command                 /opt/stopOM.sh          return_rc=True
+                            Should Be Equal As Integers     ${rc1}                  0
+    ${rc2}  ${stopMsg} =    Execute Command                 sudo ps aux | grep om
+                            Should Be Equal As Integers     ${rc2}                  0
                             Close Connection
                             Should Not Contain              ${shutdownMsg}          OzoneManagerStarter
```
Comment (adoroszlai) [inline hadoop-ozone/dist/src/main/smoketest/omha/testOMHA.robot]:
```suggestion
                             Should Not Contain              ${stopMsg}          OzoneManagerStarter
```

**Comment by elek** [comment]
This patch is required to have green acceptance test results. 

May I commit it together with the suggested changes?

**Comment by hanishakoneru** [comment]
Thank you @adoroszlai and @elek for the reviews and the suggestions. I had mistakenly uploaded an older patch. Updated it now.

**Comment by arp7** [comment]
+1 I will commit this. It looks like all of @adoroszlai 's suggestions have been incorporated. Thanks for the careful review Attila.

**Pull request merged**